packages/library/src/commonMain/kotlin/io/realm/Queryable.kt

```diff
@@ -21,6 +21,12 @@ package io.realm
 //    dependent on how the final API is going to look like.
 //  - Query could alternatively be separated into builder to await constructing new results until
 //    actually executing the query
+//  https://github.com/realm/realm-kotlin/issues/206
+/**
+ * Interface holding common query methods.
+ *
+ * Will most likely change in the future.
```

Comment:
I would remove this line. API docs just reflect the current state. Otherwise, this might add confusion for the reader